src/routes/frontpage/backers.js

```diff
@@ -0,0 +1,43 @@
+export const partners = [
+  {
+    name: 'Investa',
+    url: 'http://investa.is',
+    logo: '/logos/investa.png',
+    description: 'Investa er fjárfestingafélag sem fjárfestir í íslenskum sprotafyrirtækjum sem eiga góða möguleika á að hasla sér völl erlendis.',
+  },
+  {
+    name: 'SUT',
+    url: 'http://www.si.is/hugverk-og-thjonusta/sut/',
+    logo: '/logos/SUT.png',
+    description: 'Tilgangur stjórnar SUT er að þróa og kynna UT fyrirtæki þannig að UT verði einn áhugaverðasti iðnaður landsins.',
+  },
+  {
+    name: 'Tempo',
+    url: 'https://tempo.io',
+    logo: '/logos/tempo.png',
+    description: 'Our mission critical tools adapt to how software development and business teams work—from startup to enterprise—for real team empowerment, overview, and alignment. Collaborate, plan and schedule resources, manage budgets, and track work seamlessly within your daily workflow.',
+  },
+];
+
+export const sponsors = [
+  {
+    name: 'Frumtak',
+    url: 'https://frumtak.is',
+    logo: '/logos/frumtak.png',
+    description: 'The fund invests in early stage innovative companies that are post seed and show great potential for growth. The fund invests primarily in Iceland but is allowed to invest abroad as allowed by law in connection with its portfolio investments to ensure success in foreign markets.',
+  },
+  {
+    name: 'Kaptio',
+    url: 'https://kaptio.com',
+    logo: '/logos/kaptio.png',
+    description: 'The travel transformation platform',
+  },
+];
+export const supporters = [
+  {
+    name: 'Northstack',
+    url: 'http://northstack.is',
+    logo: '/logos/northstack.png',
+    description: "Northstack is Iceland's premier outlet for news, data, analysis, and discussion about the Icelandic startup and tech scene.",
+  },
```

Comment:
There is something weird about the mix of Icelandic and English. Easy to fix later though.